SU Split (#13)
holaluz/handbook: organization/teams/platform_team.md

# Platform team

## Mission

> Platform team mission is to help Holaluz development teams to reach development in a safe, fast and scalable way.

Reviewer: 🤔 😅 

```suggestion
Platform team mission is to help Holaluz development teams to reach production in a safe, fast and scalable way.
```

## Vision

> SU team members we help tech. dept people to develop in an efficient way applications that provide secure good services without interruptions to our customers/collaborators.

Reviewer 2: ```suggestion
Platform team members we help Technology team people to develop in an efficient way applications that provide secure good services without interruptions to our customers/collaborators.
```

## Areas of work

### Security

> Holaluz systems, network, data and applications are secure.

> * Holaluz development teams have the required tools and knowledge to adopt good security practices in they day to day work (pipelines, secure design, good practices, etc.)

Reviewer 2: ```suggestion
* Holaluz development teams have the required tools and knowledge to adopt good security practices in they day to day work (pipelines, secure design, good practices, etc.)
```
* Platform team defines security standards shared across all development teams.

### Cyberhealth

> > Holaluz systems and applications have a good performance, escalability and they are reliable.

Reviewer 2: ```suggestion
> Holaluz systems and applications have a good performance, scalability and they are reliable.
```

* Development teams know, in an automated way, when a system or application doesn't work as expected and they have all the necessary to diagnose the problem and to resolve it.

> * Holaluz Developed applications and the systems that provide those applications are resilients.

Reviewer 2: ```suggestion
* Holaluz developed applications and the systems that provide those applications are resilient.
```
